tools/clang_tidy/bin/main.dart

```diff
@@ -12,8 +12,14 @@
 //
 // User environment variable FLUTTER_LINT_ALL to run on all files.
 
+import 'dart:io' as io;
+
 import 'package:clang_tidy/clang_tidy.dart';
 
 Future<int> main(List<String> arguments) async {
-  return ClangTidy.fromCommandLine(arguments).run();
```

Comment:
Who would have thought, returning 1 from this future still exits the script with a 0 code.

https://stackoverflow.com/questions/26976550/why-main-return-value-does-not-set-exitcode-in-dart